src/app/core/theme.component.scss

```diff
@@ -54,3 +54,7 @@
     margin-bottom: 1rem;
   }
 }
+
+.displayNavbar {
```

Comment:
Navbar zasłania content strony. Także przydałoby się to poprawić.

Reply:
Done